src/cogent3/core/new_alignment.py

```diff
@@ -77,6 +81,95 @@ def assign_sequential_names(num_seqs: int, base_name: str = "seq", start_at: int
     return [f"{base_name}_{i}" for i in range(start_at, start_at + num_seqs)]
 
 
+class GapsOk:
```

Comment:
We will ignore this because we intend to rewrite the two approaches to using numpy array operations and possibly converting to `numba.jit` functions. @KatherineCaley add a comment about redesign to this class